static/src/javascripts-legacy/projects/commercial/modules/hosted/youtube.js

```diff
@@ -78,6 +78,12 @@ define([
                 } else {
                     clearTimeout(playTimer);
                 }
+
+                if (posterImage) {
+                    if (event.data === -1 || event.data === window.YT.PlayerState.PLAYING) {
```

Comment:
From the [docs](https://developers.google.com/youtube/iframe_api_reference#Events)

> When the player first loads a video, it will broadcast an unstarted (-1) event.

Unlike the others, it doesn't have a nice alias.